index.html

```diff
@@ -0,0 +1,368 @@
+            <li class="social__responsive">
+              <a href="https://twitter.com">
+                <img
+                  src="./static/imgs/twitter.png"
+                  alt="twitter"
+                  class="social__responsive__respo"
+                />
+              </a>
+            </li>
+            <li class="social__responsive">
+              <a href="https://linkedin.com">
+                <img
+                  src="./static/imgs/Linkedin icon.png"
+                  alt="Linkedin"
+                  class="social__responsive__respo"
+                />
+              </a>
+            </li>
+            <li class="social__responsive">
+              <a href="https://medium.com">
+                <img
+                  src="./static/imgs/medium.png"
+                  alt="medium"
+                  class="social__responsive__respo"
+                />
+              </a>
+            </li>
+            <li class="social__responsive">
+              <a href="https://github.com">
+                <img
+                  src="./static/imgs/github.png"
+                  alt="github"
+                  class="social__responsive__respo"
+                />
+              </a>
+            </li>
+            <li class="social__responsive">
+              <a href="https://angellist.com">
+                <img
+                  src="./static/imgs/angellist.png"
+                  alt="angellist"
+                  class="social__responsive__respo"
+                />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <!-- end sectionone -->
+    <!-- start sectiontwo -->
+    <section class="sectiontwo pd-y" id="sectiontwo">
+      <div class="container">
+        <div class="sectiontwo__container__items">
+          <div class="sectiontwo__container__items__item">
+            <div class="sectiontwo__container__items__item__responsive">
+              <img
+                src="./static/imgs/three.png"
+                alt="one"
+                class="responsive sectiontwo__container__items__item__responsive__respo"
+              />
+            </div>
+          </div>
+          <div class="sectiontwo__container__items__item">
+            <h2>Tonic</h2>
+            <ul class="first__ul">
+              <li>CANOPY</li>
+              <li class="disc">Back End Dev</li>
+              <li class="disc">2015</li>
+            </ul>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="second__ul">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button class="hover">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- end sectiontwo -->
+    <!-- start sectionthree -->
+    <section class="sectionthree pd-y" id="sectionthree">
```

Comment:
- [ ] Kindly make sure the projection section has only one `section` that is the container of all projects and each project is wrapped inside a `div`.
- [ ] Kindly avoid the deep-level wrapping of your element because they make your `HTML` file look ugly